### PR TITLE
radarr: update to v3.2.2.5080 and use custom tmp folder for updates

### LINKED ISSUE #4621: Can't update Radarr
### Setup
_Package Name:_ Radarr
_Package Version:_ 20210311-15

_NAS Model:_ DS216j
_NAS Architecture:_ armada38x
_DSM version:_ DSM 6.2.4-25556

### Expected behavior
Using Radarr's internal updater it should update to latest version

### Actual behavior
Update fails with error message

Error occurred while executing task ApplicationUpdate: No space left on device

I have also tried installing the package and reinstalling and still can't update, but I also can no longer restore a backup with a similar error

Please let me know if there is any other info I can provide. I'm a bit new to this.

### Steps to reproduce
_1._ Navigate to update tab within Radarr
_2._ Click "install latest"
_3._ 

#### Package log

[Info] InstallUpdateService: Deleting old update files 
[Info] InstallUpdateService: Downloading update 3.1.1.4954 
[Info] InstallUpdateService: Verifying update package 
[Info] InstallUpdateService: Update package verified successfully 
[Info] InstallUpdateService: Extracting Update package 
[Error] CommandExecutor: Error occurred while executing task ApplicationUpdate 

[v3.0.2.4552] System.IO.IOException: No space left on device
   at System.IO.FileStream.WriteNative(ReadOnlySpan`1 source)
   at System.IO.FileStream.WriteSpan(ReadOnlySpan`1 source)
   at System.IO.FileStream.Write(Byte[] array, Int32 offset, Int32 count)
   at ICSharpCode.SharpZipLib.Tar.TarInputStream.CopyEntryContents(Stream outputStream)
   at ICSharpCode.SharpZipLib.Tar.TarArchive.ExtractEntry(String destDir, TarEntry entry)
   at ICSharpCode.SharpZipLib.Tar.TarArchive.ExtractContents(String destinationDirectory)
   at NzbDrone.Common.ArchiveService.ExtractTgz(String compressedFile, String destination) in D:\a\1\s\src\NzbDrone.Common\ArchiveService.cs:line 110
   at NzbDrone.Common.ArchiveService.Extract(String compressedFile, String destination) in D:\a\1\s\src\NzbDrone.Common\ArchiveService.cs:line 36
   at NzbDrone.Core.Update.InstallUpdateService.InstallUpdate(UpdatePackage updatePackage) in D:\a\1\s\src\NzbDrone.Core\Update\InstallUpdateService.cs:line 133
   at NzbDrone.Core.Update.InstallUpdateService.Execute(ApplicationUpdateCommand message) in D:\a\1\s\src\NzbDrone.Core\Update\InstallUpdateService.cs:line 275
   at NzbDrone.Core.Messaging.Commands.CommandExecutor.ExecuteCommand[TCommand](TCommand command, CommandModel commandModel) in D:\a\1\s\src\NzbDrone.Core\Messaging\Commands\CommandExecutor.cs:line 86
   at CallSite.Target(Closure , CallSite , CommandExecutor , Object , CommandModel )
   at System.Dynamic.UpdateDelegates.UpdateAndExecuteVoid3[T0,T1,T2](CallSite site, T0 arg0, T1 arg1, T2 arg2)
   at CallSite.Target(Closure , CallSite , CommandExecutor , Object , CommandModel )
   at NzbDrone.Core.Messaging.Commands.CommandExecutor.ExecuteCommands() in D:\a\1\s\src\NzbDrone.Core\Messaging\Commands\CommandExecutor.cs:line 44

### LINKED ISSUE #4755: Cannot update Radarr
### Setup
_Package Name:_ Radarr
_Package Version:_ 3.0.2.4552

_NAS Model:_ DS220j
_NAS Architecture:_ rtd1296 (armv8)
_DSM version:_ 6.2.4-25556

### Expected behavior
Trying to update to latest version

### Actual behavior
Error (see log details)

#### Package log
|Error|CommandExecutor|Error occurred while executing task ApplicationUpdate

[v3.0.2.4552] System.IO.IOException: No space left on device
at System.IO.FileStream.WriteNative(ReadOnlySpan`1 source)
   at System.IO.FileStream.WriteSpan(ReadOnlySpan`1 source)
   at System.IO.FileStream.Write(Byte[] array, Int32 offset, Int32 count)
   at ICSharpCode.SharpZipLib.Tar.TarInputStream.CopyEntryContents(Stream outputStream)
   at ICSharpCode.SharpZipLib.Tar.TarArchive.ExtractEntry(String destDir, TarEntry entry)
   at ICSharpCode.SharpZipLib.Tar.TarArchive.ExtractContents(String destinationDirectory)
   at NzbDrone.Common.ArchiveService.ExtractTgz(String compressedFile, String destination) in D:\a\1\s\src\NzbDrone.Common\ArchiveService.cs:line 110
   at NzbDrone.Common.ArchiveService.Extract(String compressedFile, String destination) in D:\a\1\s\src\NzbDrone.Common\ArchiveService.cs:line 36
   at NzbDrone.Core.Update.InstallUpdateService.InstallUpdate(UpdatePackage updatePackage) in D:\a\1\s\src\NzbDrone.Core\Update\InstallUpdateService.cs:line 133
   at NzbDrone.Core.Update.InstallUpdateService.Execute(ApplicationUpdateCommand message) in D:\a\1\s\src\NzbDrone.Core\Update\InstallUpdateService.cs:line 275
   at NzbDrone.Core.Messaging.Commands.CommandExecutor.ExecuteCommand[TCommand](TCommand command, CommandModel commandModel) in D:\a\1\s\src\NzbDrone.Core\Messaging\Commands\CommandExecutor.cs:line 118
   at System.Dynamic.UpdateDelegates.UpdateAndExecuteVoid3[T0,T1,T2](CallSite site, T0 arg0, T1 arg1, T2 arg2)
   at NzbDrone.Core.Messaging.Commands.CommandExecutor.ExecuteCommands() in D:\a\1\s\src\NzbDrone.Core\Messaging\Commands\CommandExecutor.cs:line 44

--------

Sorry guys total newbie with NAS, looks as similar error to Jackett. Is it something about DS220j low specs?? I have plenty space left on HDD -1.6GB. Thank you

### PR DESCRIPTION
**Pull request by hgy59**
_Motivation:_  Radarr internal update seems to run out of space
_Linked issues:_  closes #4755, closes #4621

### Checklist
- [x] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully

### Remarks
- This fixes package folders for DSM7.

### CODE REVIEW

File: spk/radarr/Makefile [hunk 1]
```diff
@@ -35,6 +35,9 @@ WIZARDS_DIR = src/wizard/
 
 POST_STRIP_TARGET = radarr_extra_install
 
+# use alternate TMPDIR as /tmp might be too small.
+USE_ALTERNATE_TMPDIR = 1
```
Comment (publicarray) [inline spk/radarr/Makefile:39]: I think we should also add this to the other arr* packages, inc. Jackett https://github.com/SynoCommunity/spksrc/issues/4754

**Review by publicarray — APPROVED** [review]

**Comment by publicarray** [comment]
For Reference on a NAS with 2GB RAM (virtual machine) `/tmp` is 1GB - DSM7

```
$ df -h
Filesystem      Size  Used Avail Use% Mounted on
/dev/sda1       2.0G  1.1G  760M  59% /
devtmpfs        998M     0  998M   0% /dev
tmpfs          1002M   68K 1002M   1% /dev/shm
tmpfs          1002M   12M  990M   2% /run
tmpfs          1002M     0 1002M   0% /sys/fs/cgroup
tmpfs          1002M  1.1M 1001M   1% /tmp
/dev/sdb1        18G  2.6G   15G  15% /volume1
```

For Reference on a NAS with 4GB RAM (virtual machine) `/tmp` is 2GB - DSM6

```
$ df -h
Filesystem      Size  Used Avail Use% Mounted on
/dev/md0        2.3G  1.1G  1.2G  47% /
none            2.0G     0  2.0G   0% /dev
/tmp            2.0G  1.2M  2.0G   1% /tmp
/run            2.0G  3.1M  2.0G   1% /run
/dev/shm        2.0G  8.0K  2.0G   1% /dev/shm
none            4.0K     0  4.0K   0% /sys/fs/cgroup
cgmfs           100K     0  100K   0% /run/cgmanager/fs
/dev/md2         11G  2.5G  8.0G  24% /volume1
/dev/md3         25G   11G   14G  45% /volume2
tmpfs           1.0T  2.0G 1022G   1% /dev/virtualization

```

**Comment by houndtt** [comment]
hey team, will this be getting merged soon? eager to see the new package

**Review by ymartin59 — APPROVED** [review]

**Comment by mreid-tt** [comment]
hey team, any timeline on this commit being accepted and published?

**Pull request merged**